docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -0,0 +1,64 @@
+= Parameters
+
+The parent key for all of the following parameters is `metallb`.
+
+
+== `namespace`
+
+[horizontal]
+type:: string
+default:: `syn-metallb`
+
+
+
+== `speaker.secretkey`
+
+[horizontal]
+type:: string
+default:: Vault reference
+
+A binary secret key encoded base64.
+It can be generated using the command `openssl rand -base64 128`.
+
+== `addresses`
+
+[horizontal]
+type:: list
+default:: null
+
+A list of IPs (for example `10.128.1.200/32`) used as transferable virtual IPs.
```

Comment:
MetalLB actually wants CIDRs, and it's perfectly acceptable to provide

```
addresses:
- 10.128.1.192/27
```

if the VIP pool is allocated as that CIDR. When configured this way, MetalLB will allocate an IP from the CIDR whenever a new service of type `LoadBalancer` is created.

You probably want to note that the component creates an address pool called "default", if `metallb.addresses` is configured.